packages/@uppy/provider-views/src/ProviderView/ProviderView.jsx

```diff
@@ -280,6 +280,7 @@ export default class ProviderView extends View {
             if (!this.plugin.uppy.checkIfFileAlreadyExists(id)) {
               newFiles.push(fileInFolder)
               numNewFiles++
+              this.setLoading(this.plugin.uppy.i18n('addedNumFiles', { numFiles: numNewFiles }))
```

Comment:
Why do we pass it to `setLoading`? Can't we inline the `i18n` in JSX?

Reply:
In the UI component, can't we do something like this 

```jsx
{loading ? this.plugin.uppy.i18n('addedNumFiles', { numFiles: numNewFiles }) : null}
```

And only have `setLoading` as `boolean` toggle?

Reply:
Yes but then we need to have one loading state variable to indicate loading *without text* and another state variable for loading *with text*

Reply:
Can't we always show it? When wouldn't we want to show it?

Reply:
I thought it doesn't make any sense to show it when we are simply adding a list of files, or when the user browses through folders, I don't think it makes sense to show "adding X files", e.g. this function: https://github.com/transloadit/uppy/blob/d67a4a59ee4378d637a7dee9fbda733c01decd0c/packages/%40uppy/provider-views/src/ProviderView/ProviderView.jsx#L105